### DISCUSSION TITLE
Retroactively fixup GPTempDownload filenames and metadata in iCloud

### DISCUSSION THREAD

**Discussion by adamel** [fxstein/GoProX#54] [Q&A]
Hi,

I made the mistake of importing photos from my GoPro 11 to my iPhone over WiFi using the QUIK app.

1. Is there any way to use GoProX on the photos synced to the Photos app on my MBP and retroactively fix the filenames and other meta data, or do I have to delete all the media from Photos and re-import it from the SD-card using GoProX?
2. Is there any sane way to import media from GoPro 11 over WiFi to my iPhone and have correct filenames and meta data, or do I have to wait until I get home and plug the SD-card into the MBP and use GoProX there?

**Comment by fxstein** [comment]
Hi,

Welcome to the world of GoPro Quik! You're describing the reason for me writing goprox in the first place. Quik literally destroys metadata and messes up files along the way.

Unfortunately, I have entirely abandoned Quik for anything but basic settings of cameras, but even then, the labs firmware with QR codes is much more reliable. 

Unfortunately, the only way I know how to deal with it is to plug the Sd cards into a reader and do the initial import and processing with goprox. I then import the processed files into Photos.

You can use Quik to preview the material on the road but never use it to import. I usually bring my MBP on longer trips to unload the sd cards in the evening or after hours. For the rest, I bring extra sd cards.

Hope this helps!
